File: Application/Services/Operations/Finances/Dtos/MonthlyExpenses/MonthlyFixedExpensePaymentDto.cs

using System;
using Application.Services.Operations.Finances.Dtos.InheritanceDto;
using Application.Services.Operations.Finances.Dtos.PixExpenses;


namespace Application.Services.Operations.Finances.Dtos.MonthlyExpenses
{
    public class MonthlyFixedExpensePaymentDto
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int CompanyId { get; set; }
        public int BankAccountId { get; set; }
        public int? CardId { get; set; }
        public int? PixId { get; set; }
        public string OthersPaymentMethods { get; set; }
        public PixExpenseDto PixExpense { get; set; }
        public decimal Price { get; set; }
        public decimal Interest { get; set; }
        public DateTime WasPaid { get; set; }
        public string Document { get; set; }
    }
}

File: Views/src/components/financial/components/monthly-fixed-expenses/dto/monthly-fixed-expense-payment-dto.ts
import { PixExpenseDto } from "../../pix-expenses/dto/pix-expense-dto";

export class MonthlyFixedExpensePaymentDto {
    id: number;
    userId: number;
    companyId: number;
    bankAccountId: number;
    cardId: number;
    pixId: number;
    othersPaymentMethods: string;
    pixExpense: PixExpenseDto;
    expires: Date;
    price: number;
    interest: number;
}
